### PR TITLE
show icons for the know server types

### LINKED ISSUE #292: use different icons for known server types
we should use specific icons for the different servers that we offer so that users immediately recognize the kind of server.

### PR DESCRIPTION
**Pull request by adietish**
fixes #292

### CODE REVIEW

**Comment by adietish** [comment]
Here are the icon that this provides: image

**Comment by adietish** [comment]
testPR

File: src/serverIcon.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+export class ServerIcon {
+
+  public static get(serverType: Protocol.ServerType): Uri {
+      if (!serverType
+          || !serverType.id) {
+              return null;
+      }
+
+      return Uri.file(path.join(__dirname, '../../images/' + this.getFilename(serverType)));
```
Comment (jrichter1) [inline src/serverIcon.ts]: Let's use ```path.join``` properly, shall we?
```typescript
path.join(__dirname, '..', '..', 'images', this.getFilename(...))
```

File: test/serverIcons.test.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
+/*-----------------------------------------------------------------------------------------------
+ *  Copyright (c) Red Hat, Inc. All rights reserved.
+ *  Licensed under the EPL v2.0 License. See LICENSE file in the project root for license information.
+ *-----------------------------------------------------------------------------------------------*/
+
+import * as chai from 'chai';
+import * as sinonChai from 'sinon-chai';
```
Comment (jrichter1) [inline test/serverIcons.test.ts]: no need for sinonChai without sinon

**Comment by adietish** [comment]
@jrichter1 given that indentation is inconsitent, dare to make sure we have it done automatically? https://github.com/redhat-developer/vscode-adapters/issues/294

**Pull request merged**